core/src/main/java/tc/oc/pgm/api/map/Gamemode.java

```diff
@@ -0,0 +1,44 @@
+package tc.oc.pgm.api.map;
+
+public enum Gamemode {
+  ATTACK_DEFEND("ad", "Attack/Defend"),
```

Comment:
```suggestion
  ATTACK_DEFEND("ad"),
```
Since these are simply translated in the string files, they're not needed in the code, you can remove all references to name in general